### PR TITLE
Citation file

### PR DESCRIPTION
**Pull request by rlskoeser**
citation CFF generated based on Zenodo doi and modified slightly

@ColeDCrawford @jdamerow @maltevogl @robcast please check that you are listed as you would like to be (name is correct; add/update affiliation and ORCiD if desired). You can use the suggest changes option in a PR comment or just add a message with any changes you'd like made.

Please also let me know if you'd like any changes made to the DOI record on Zenodo beyond any affiliation/orcid added here. https://zenodo.org/doi/10.5281/zenodo.11068867

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  orcid: https://orcid.org/0000-0002-8762-8057
+- given-names: Cole
+  family-names: Crawford
+  affiliation:
+- given-names: Julia
+  family-names: Damerow
+  affiliation:
+- given-names: Malte
+  family-names: Vogl
+  affiliation:
```
Comment (maltevogl) [inline CITATION.cff:20]: Affiliation Max-Planck-Institute of Geoanthropology
ORCID: 0000-0002-2683-6610

**Review by maltevogl — DISMISSED** [review]
Looks good to me, I added my aff and orcid.

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- given-names: Rebecca Sutton
+  family-names: Koeser
+  affiliation: Center for Digital Humanities, Princeton University
+  orcid: https://orcid.org/0000-0002-8762-8057
+- given-names: Cole
+  family-names: Crawford
+  affiliation:
+- given-names: Julia
+  family-names: Damerow
+  affiliation:
```
Comment (jdamerow) [inline CITATION.cff:17]:
```suggestion
  affiliation: Arizona State University
  orcid: https://orcid.org/0000-0002-0874-0092
```

**Review by jdamerow — APPROVED** [review]

**Comment by rlskoeser** [comment]
@robcast please review the citation and let me know if you want any changes to your affiliation or want to add an ORCiD

**Review by robcast — APPROVED** [review]

**Pull request merged**